src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -88,6 +91,24 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         model.setPerson(personToEdit, editedPerson);
+
+        if (personToEdit.getStatus().toString().equals(Status.NEGATIVE)
+                && editedPerson.getStatus().toString().equals(Status.POSITIVE)) {
+
+            List<Person> filteredByClassCodeList = studentList.stream()
+                    .filter(student -> student.getClassCode().toString().equals(editedPerson.getClassCode().toString())
+                            && !student.isSamePerson(editedPerson))
+                    .collect(Collectors.toList());
+
+            for (int i = 0; i < filteredByClassCodeList.size(); i++) {
+                Person currentPerson = filteredByClassCodeList.get(i);
+                EditPersonDescriptor tempDescriptor = new EditPersonDescriptor();
+                tempDescriptor.setStatus(new Status(Status.CLOSE_CONTACT));
+                Person editedPersonStatus = createEditedPerson(currentPerson, tempDescriptor);
+                model.setPerson(currentPerson, editedPersonStatus);
+            }
+        }
```

Comment:
Should we also handle the case where a student's status has changed from positive to negative? When a student has recovered, their classmates' status should be negative instead of close contact if there is no more positive student in the class.